### PR TITLE
DOCS-4437: Update CSPM Security Findings Explorer 

### PR DESCRIPTION
**Pull request by drichards-87**
### Motivation
DOCS-4437

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/cspm/findings.md [hunk 1]
```diff
@@ -19,47 +19,33 @@ Cloud Security Posture Management is not available in this site.
...
-- Use the “time selector” to explore your security configuration posture at any point in the past.
-
-In addition to reviewing and responding to findings, you can set notifications for failed findings, and configure signals to correlate and triage misconfigurations in the same view as real-time threats generated by [Cloud SIEM][2] and [Cloud Workload Security][3]. This allows users to accelerate investigations as the root-cause for many of today’s cloud breaches are misconfigured services that have been nefariously exploited.
-
-{{< img src="security/cspm/findings/findings-time-window.png" alt="Set a findings time window using the dropdown" style="width:65%;">}}
+In addition to reviewing and responding to findings, you can set notifications for failed findings, and configure signals to correlate and triage misconfigurations in the same view as real-time threats generated by [Cloud SIEM][2] and [Cloud Workload Security][3]. This enables you to accelerate investigations as the root-cause for many of today's cloud breaches are misconfigured services that have been exploited by attackers.
```
Comment (cswatt) [inline content/en/security/cspm/findings.md:30]:
```suggestion
In addition to reviewing and responding to findings, you can set notifications for failed findings, and configure signals to correlate and triage misconfigurations in the same view as real-time threats generated by [Cloud SIEM][2] and [Cloud Workload Security][3]. This enables you to accelerate investigations, as the root causes for many of today's cloud breaches are misconfigured services that have been exploited by attackers.
```

File: content/en/security/cspm/findings.md [hunk 2]
```diff
@@ -19,47 +19,33 @@ Cloud Security Posture Management is not available in this site.
...
+A finding is the primary primitive for a rule evaluation against a resource. Every time a resource is evaluated against a rule, a finding is generated with a **Pass** or **Fail** status. Resources are evaluated in increments between 15 minutes and four hours (depending on type). Datadog generates new findings as soon as a scan is completed, and stores a complete history of all findings for the past 15 months so they are available in case of an investigation or audit.
 
-{{< img src="security/cspm/findings/evaluation-fail.png" alt="Filtering by evaluation fail" style="width:100%;">}}
+## Explore your cloud misconfigurations
 
-The side panel shows details of each resource that has been evaluated by the rule.
+Findings are displayed on the [Security Findings Explorer][1]. Aggregate findings by rule using the **Group by** filters and query search bar. For example, filtering by `evaluation:fail` narrows the list to all detection rules that have issues that need to be addressed. Findings can also be aggregated by resource to rank order resources that have the most failed findings so you can prioritize remediation.
```
Comment (cswatt) [inline content/en/security/cspm/findings.md:38]:
```suggestion
Findings are displayed on the [Security Findings Explorer][1]. Aggregate findings by rule using the **Group by** filters and query search bar. For example, filtering by `evaluation:fail` narrows the list to all detection rules that have issues that need to be addressed. Findings can also be aggregated by resource to rank resources that have the most failed findings so you can prioritize remediation.
```

the "order" looks like a typo to me--unless "order resources" are a thing? in which case ignore this

File: content/en/security/cspm/findings.md [hunk 3]
```diff
@@ -19,47 +19,33 @@ Cloud Security Posture Management is not available in this site.
...
+{{< img src="security/cspm/findings/posture-management-overview-2.png" alt="An overview of the Posture Management Findings page" style="width:100%;">}}
 
-Findings can also be aggregated by resource to rank order resources that have failed the most rule evaluations so you can prioritize remediation.
+Select a finding to view the resources that have been evaluated by the rule, the rule description, its framework or industry benchmark mapping, and suggested remediation steps. Click **View Finding** to see details about the misconfigured resource.
 
-{{< img src="security/cspm/findings/eval-fail-group-by-resource.png" alt="Group and aggregate by resource in search" style="width:100%;">}}
+{{< img src="security/cspm/findings/finding-side-panel.png" alt="Ranked order resources in the side panel" style="width:65%;">}}
 
-The side panel lists detection rules that were evaluated against the resource, some of which you may choose to be addressed to improve your security configuration posture.
+Group by **Resources** on the Security Findings Explorer and select a resource to see the full list of detection rules that were evaluated against the resource, along with their status.
```
Comment (cswatt) [inline content/en/security/cspm/findings.md:46]:
```suggestion
Group by **Resources** on the Security Findings Explorer and select a resource to see the full list of detection rules that were evaluated against the resource, along with their statuses.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
small nits

**Comment by drichards-87** [comment]
Thanks @cswatt! I added your changes to the PR.

**Review by cswatt — APPROVED** [review]

**Pull request merged**